### ISSUE TITLE
After upgrading to 0.12.1 the output file doesn't work

### ISSUE THREAD

**Issue by antonmoiseev** [systemjs/builder#213]
I've just upgraded to SystemJS Builder 0.12.1 from 0.10.6 and after running `builder.build()` the output file doesn't work when I launch the app. No errors neither when I run the build script nor when I open the app. I've noticed one difference with the old version of the output file, the old one has `"format register";` statement at the very top of the file, the new one doesn't.

Here is how I invoke `build()`:
```js
let builder = new Builder();
builder.loadConfig('./app/config.js')
  .then(() => {
    builder.config({baseURL: './app'});
    return builder.build('main.js', 'dist/main.js');
  })
  .catch(console.log);
```
The meaningful part of `config.js`:
```js
System.config({
  'baseURL': '/',
  'defaultJSExtensions': true,
  'transpiler': 'traceur',
  'paths': {
    'github:*': 'jspm_packages/github/*'
  },
  'map': {
    'html': 'plugins/html.js'
  }
});
```

**Comment by guybedford** [comment]
This may be an issue with the baseURL. In the latest SystemJS the baseURL needs to be set before any other configurations and it must be set first.

**Comment by antonmoiseev** [comment]
Updated my build script:
```js
let builder = new Builder({baseURL: './app'}); // 1. Init baseURL as part of the constructor call
builder.loadConfig('./app/config.js') // 2. Removed baseURL from config.js
  .then(() => builder.build('main.js', 'dist/main.js'))
  .catch(console.log);
```
Statement `"format register"` still isn't there. I'm just getting familiar with SystemJS. Is it the actual issue, should `"format register"` be a part of the output file? Is there a workaround, e.g. using different module format for the output file?

**Comment by guybedford** [comment]
@antonmoiseev did you try adding `"format register"` into the bundle manually and did that make it work here?

**Comment by antonmoiseev** [comment]
Right, sorry, forgot to mention. I've added `"format register"` manually and everything works fine.

**Comment by guybedford** [comment]
@antonmoiseev ok thanks for clarifying. Does your bundle file contain `System.register` statements? Or are all your modules AMD perhaps?

**Comment by antonmoiseev** [comment]
Yes, there are `System.register` statements. The app uses ES6 modules, but there is a custom plugin that translates loaded content to a CommonJS module. BTW, here is a sample app that reproduces this behavior - https://github.com/antonmoiseev/polymer-upside-down:
1. `npm run deps` to install all dependencies
2. `gulp` to build the bundle
3. `gulp serve:dist` to start bundled version of the app. After starting the server you can manually add `"format register";` to dist/main.js.

**Comment by guybedford** [comment]
Thanks, I've added a fix for this in SystemJS at https://github.com/systemjs/systemjs/commit/4910332ed323d86adb573db52c5aaa2dd1c8050c. Will post a release soon and update here.

**Comment by guybedford** [comment]
This is released in SystemJS 0.18.4.

**Comment by antonmoiseev** [comment]
Confirmed, works fine now. Thank you! :+1:
